features/support/steps/network.ts

```diff
@@ -178,7 +177,12 @@ Then('{string} should have received only these lobbies', function (this: World,
         delete lobby[key]
       }
     })
-    assert.notStrictEqual(lobby, row)
```

Comment:
Such a stupid name. It should be named `strictNotEqual` instead, as that's what it does.

Reply:
I don't want to talk about it. 😝